README.md

```diff
@@ -9,7 +9,7 @@ Identity generation in Go for the [Tanker SDK](https://docs.tanker.io/latest/).
 ## Installation
 
 ```bash
-go get github.com/TankerHQ/identity-go/v3/identity
+go get github.com/TankerHQ/identity-go/v3
```

Comment:
So this breaks the API... Shouldn't we make this a v4? But then I think we'll need to bump all other identity SDKs...

Another option is to put deprecated functions in `v3/identity` that call the ones in `v3`. But I think this is not allowed in golang, so we might need to move the whole implementation in `internal` and call it with deprecated functions in `v3/identity` and normal functions in `v3`

Reply:
does it really breaks the api ? Does it just break user code by having to fix the module path ?

Reply:
Why wouldn't we be able to implement your solution @blastrock ? I think `v3/identity` can completely call `v3` as long as there's no dependency cycle
I will have many other PRs to pass for refactoring and such, most will be pretty small, others are likely to raise questions as to how to handle versioning etc. Maybe all of this is going into v4 indeed. That would also allow for broader changes. I don't know how supposed to work around that..

Reply:
You are allowed to break the API. But if already know that the incoming patch/release will break it multiple times, it's best to do that only once, by doing one final release. 
Also as stated before, if you make a breaking change, you must release all the other identity sdk.

Reply:
Okay, to resume what was discussed outside:
The migration plan is
```
Minor version 3.2:
add new functions
v3/identity -> v3 new functions
deprecate v3/identity

Major version 4:
remove v3/identity
```

This PR precisely is the first of all steps required, new functions will probably arrive at the end of the migration process, putting an end to it. So I'm adding the mapping from `v3/identity` to `v3` and will update the names later.

Reply:
I've made the mapping, the go.mod file contains a `replace` directive that refers to a commit that will be replaced at the end of the process when the minor version is released, and it will point to the new minor.